rma_account/tests/test_rma.py

```diff
@@ -0,0 +1,395 @@
+        self.make_supplier_rma = self.env["rma.order.line.make.supplier.rma"]
+        self.rma_add_stock_move = self.env['rma_add_stock_move']
+        self.stockpicking = self.env['stock.picking']
+        self.rma = self.env['rma.order']
+        self.rma_line = self.env['rma.order.line']
+        self.rma_op = self.env['rma.operation']
+        self.rma_cust_replace_op_id = self.env.ref(
+            'rma.rma_operation_customer_replace')
+        self.rma_sup_replace_op_id = self.env.ref(
+            'rma.rma_operation_supplier_replace')
+        self.product_id = self.env.ref('product.product_product_4')
+        self.product_id.product_tmpl_id.categ_id.\
+            property_stock_account_input_categ_id =\
+            self.env.ref('account.data_account_type_receivable').id
+        self.product_id.product_tmpl_id.categ_id.\
+            property_stock_account_output_categ_id =\
+            self.env.ref('account.data_account_type_expenses').id
+        self.product_1 = self.env.ref('product.product_product_25')
+        self.product_2 = self.env.ref('product.product_product_30')
+        self.product_3 = self.env.ref('product.product_product_33')
+        self.uom_unit = self.env.ref('product.product_uom_unit')
+        # assign an operation
+        self.product_1.write(
+            {'rma_customer_operation_id': self.rma_cust_replace_op_id.id,
+             'rma_supplier_operation_id': self.rma_sup_replace_op_id.id})
+        self.product_2.write(
+            {'rma_customer_operation_id': self.rma_cust_replace_op_id.id,
+             'rma_supplier_operation_id': self.rma_sup_replace_op_id.id})
+        self.product_3.write(
+            {'rma_customer_operation_id': self.rma_cust_replace_op_id.id,
+             'rma_supplier_operation_id': self.rma_sup_replace_op_id.id})
+        self.partner_id = self.env.ref('base.res_partner_12')
+        self.stock_location = self.env.ref('stock.stock_location_stock')
+        self.stock_rma_location = self.env.ref('rma.location_rma')
+        self.customer_location = self.env.ref(
+            'stock.stock_location_customers')
+        self.supplier_location = self.env.ref(
+            'stock.stock_location_suppliers')
+        self.product_uom_id = self.env.ref('product.product_uom_unit')
+        products2move = [(self.product_1, 3), (self.product_2, 5),
+                         (self.product_3, 2)]
+        self.rma_customer_id = self._create_rma_from_move(
+            products2move, 'customer', self.env.ref('base.res_partner_2'),
+            dropship=False)
+
+    def _create_picking(self, partner):
+        return self.stockpicking.create({
+            'partner_id': partner.id,
+            'picking_type_id': self.env.ref('stock.picking_type_in').id,
+            'location_id': self.stock_location.id,
+            'location_dest_id': self.supplier_location.id
+            })
+
+    def _create_rma_from_move(self, products2move, type, partner, dropship,
```

Comment:
I'm not sure why this is tested in all the modules, IMHO it's necessary just once, in RMA module